common/xrdp_client_info.h

```diff
@@ -38,7 +38,8 @@ struct monitor_info
     int bottom;
     int flags;
 
-    /* From 2.2.2.2.1 DISPLAYCONTROL_MONITOR_LAYOUT */
+    /* From [MS-EDISP] 2.2.2.2.1 DISPLAYCONTROL_MONITOR_LAYOUT, or
```

Comment:
https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpedisp/d2954508-f487-48bc-8731-39743e0854a9

Should we use the full name here: `MS-RDPEDISP`

Reply:
Yes we should - I'll fix it.

Reply:
Done